File: scripts/audit/fixtures/PAN003-locale-parity/good/scripts/lib/locales.py
"""Fixture (PAN003-locale-parity / good): Python locale SoT that AGREES with the
sibling src/lib/locales.ts. The audit execs this module, so it must import."""

LOCALES: tuple[str, ...] = ("ru", "en")
DEFAULT_LOCALE: str = "ru"

File: scripts/audit/fixtures/PAN003-locale-parity/good/src/lib/locales.ts
// Fixture (PAN003-locale-parity / good): TS locale SoT that AGREES with the
// sibling scripts/lib/locales.py — the cross-language audit must stay silent.
export const LOCALES = ["ru", "en"] as const;
export type Locale = (typeof LOCALES)[number];
export const DEFAULT_LOCALE: Locale = "ru";
